finetuner/data.py

```diff
@@ -39,12 +42,24 @@ class CSVOptions:
     convert_to_blob: bool = True
 
 
-def build_dataset(
+def build_finetuning_dataset(
     data: Union[str, TextIO, DocumentArray],
     model: str,
     csv_options: Optional[CSVOptions] = None,
 ) -> Union[str, DocumentArray]:
+    """If data has been provided as a CSV file, the given CSV file is parsed
+        and a :class:`DocumentArray` is created.
+
+    :param data: The input data provided by the user, either a path to a CSV file,
```

Comment:
dont document these functions with arguments. They are internal. Just a docstring is fine